src/bot/commands/utilities/poll.ts

```diff
@@ -0,0 +1,59 @@
+import {Command, Input} from '@api';
+import {TextChannel} from 'discord.js';
+import {UserPoll} from "@libraries/utilities/create_poll";
+import {Emoji} from "@libraries/emoji";
+
+export class Poll extends Command {
+    constructor() {
+        super({
+            name: 'poll',
+            description: 'Lets members vote on between 2-6 options',
+            arguments: [
+                {
+                    name: 'question',
+                    constraint: 'string',
+                    required: true,
+                },
+                {
+                    name: 'choices',
```

Comment:
```suggestion
                    name: 'choices',
                    description: 'A list of choices separated by commas.',
```

And this really needs to be here.